docs/GoogleDrive-batchsink.md

```diff
@@ -20,22 +20,24 @@ Is optional. In the case it is not set files have randomly generated 16-symbols
 All MIME types are supported except [Google Drive types](https://developers.google.com/drive/api/v3/mime-types).
 Is optional. In the case it is not set Google API will try to recognize file's MIME type automatically.
 
-**Directory Identifier:** Identifier of the destination folder.
+**Directory Identifier:** Identifier of the destination folder. This is everything that comes after “folders/” in the 
+URL. For example, if the URL was “https://drive.google.com/drive/folders/1dyUEebJaFnWa3Z4n0BFMVAXQ7mfUH11g”, then the 
+Directory Identifier would be “1dyUEebJaFnWa3Z4n0BFMVAXQ7mfUH11g”.
 
 ### Authentication
 
-**Authentication Type:** Type of authentication used to access Google API. 
+**Authentication Type:** Type of authentication used to access Google API. Make sure Google Drive Api is enabled.
 OAuth2 and Service account types are available.
 
 #### OAuth2 Properties
```

Comment:
Can you add some details on how to generate oauth token?

Reply:
Acknowledged